server/app.js

```diff
@@ -29,7 +33,7 @@ const child_process = require("child_process");
 const WatsonVisRecSetup= require('./lib/watson-visRec-setup');
 
 const visual_recognition = watson.visual_recognition({
-    api_key: "",
+    api_key: credentials.api_key,
```

Comment:
If you remove this the SDK should read your api_key from the VCAP watson_vision_combined... or from process.env.VISUAL_RECOGNITION_API_KEY

Reply:
ok, I'll test...